jax/_src/lib/xla_bridge.py

```diff
@@ -257,6 +257,22 @@ def make_gpu_client(*, platform_name, visible_devices_flag):
 if iree is not None:
   register_backend_factory("iree", iree.iree_client_factory, priority=-100)
 
+# Register PJRT C API plugins based on environment variables.
+def _initialize_plugin_backends():
+  def make_factory(name, path):
+    def factory():
+      xla_client._xla.load_pjrt_plugin(name, path)
+      return xla_client._xla.get_c_api_client(name)
+    return factory
+
+  pjrt_plugins = xla_client._get_pjrt_plugin_names_and_library_paths()
+  for plugin_name, library_path in pjrt_plugins.items():
+    register_backend_factory(
+      plugin_name, make_factory(plugin_name, library_path))
+
+if hasattr(xla_client, "_get_pjrt_plugin_names_and_library_paths"):
```

Comment:
@skye I know this change is a short term work around. Do we want to expose _get_pjrt_plugin_names_and_library_paths (i.e. remove "_" and add to xla_client.pyi?